src/util/calculateCliffDropTargetHeight.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Calculates the height of a drop target at the cliff, decreasing with depth depth. This makes it easier to drop a thought at the desired level. Returns height in ems.
+ */
+const calculateCliffDropTargetHeight = ({ cliff = 0, depth }: { cliff?: number; depth?: number }) => {
+  if (depth === undefined) return 0
+
+  // Calculate dynamic height: increase as depth decreases
+  const baseHeight = 1
+  const heightIncrease = 0.5
+  const dropTargetHeight = baseHeight + heightIncrease * ((-cliff || depth) - depth)
```

Comment:
This is just a minor suggestion, but I believe the functionality of this line would be clearer with a ternary instead of `(-cliff || depth) - depth)`:

```suggestion
  const dropTargetHeight = cliff === 0
    ? baseHeight
    : baseHeight + heightIncrease * (-cliff - depth)
```

Reply:
Thanks, I updated the code.